hs_core/views/__init__.py

```diff
@@ -117,7 +117,9 @@ def change_quota_holder(request, shortkey):
     except PermissionDenied:
```

Comment:
@pkdash Can you point out an example for unit test for a view function for my reference to save me time to look for it myself just to speed things up? I will get it added first thing tomorrow morning so that this PR can make into release 1.11.